### ISSUE TITLE
emit translate-c warnings as comments rather than (omitted by default) messages to stderr

### ISSUE THREAD

**Issue by andrewrk** [ziglang/zig#2259]
Right now you have to do this to observe warnings from translate-c:

#### foo.h
```c
struct Foo {
    unsigned a : 1;
    unsigned b : 7;
};
```

#### terminal
```
$ zig translate-c foo.h --verbose-cimport
test.h:2:14: warning: struct Foo demoted to opaque type - has bitfield
pub const struct_Foo = @OpaqueType();
...
```

This is not obvious, and results in confusion such as in #2257. Further, the warnings are not well connected to `@cImport` expressions.

Instead, the warnings should be in the AST itself, as comments, perhaps even doc comments describing the type that could not be translated.

This wasn't my idea; it came from someone in the Zig community but I don't remember where.

Note that modifications to translate-c are subject to #1964. If working on a patch for this issue, please try to use the C API where possible (and maybe even help the conversion effort).

**Comment by andrewrk** [comment]
This can be done in the self-hosted version only (See #1964).

**Comment by Vexu** [comment]
Closed by #4025
